editor/app/src/main/java/ca/ubc/eml/soiltopargraphy/editor/ui/flag/FlagListFragment.java

```diff
@@ -33,22 +30,15 @@ public static FlagListFragment newInstance() {
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
                              @Nullable Bundle savedInstanceState) {
-
```

Comment:
This code is still good! You should bring it back 😃 
Just change line 46 to return mView and you're set.